maps/src/maputils.cxx

```diff
@@ -179,6 +198,39 @@ boost::python::tuple GetRaDecMap(G3SkyMapConstPtr m)
 }
 
 
+G3SkyMapPtr GetRaDecMask(G3SkyMapConstPtr m, double ra_left, double ra_right,
+    double dec_bottom, double dec_top)
+{
+	G3SkyMapPtr mask = m->Clone(false);
+	mask->weighted = false;
+	mask->units = G3Timestream::None;
+	mask->pol_type = G3SkyMap::None;
+	mask->SetPolConv(G3SkyMap::ConvNone);
+
+	if (ra_left < 0)
+		ra_left += 360 * G3Units::deg;
+	if (ra_right < 0)
+		ra_right += 360 * G3Units::deg;
+
+	for (size_t i = 0; i < m->size(); i++) {
+		std::vector<double> radec = m->PixelToAngle(i);
+		double ra = radec[0];
+		if (ra < 0)
+			ra += 360 * G3Units::deg;
```

Comment:
Same issue here. There's also a potential issue with wrapping on the right, for RA > 2 pi.